include/seqan3/alignment/pairwise/alignment_result.hpp

```diff
@@ -190,7 +199,7 @@ class alignment_result
 
     /*!\brief Returns the actual alignment, i.e. the base pair matching.
      * \return At least two aligned sequences, which represent the alignment.
-     * \attention This function with fail the compilation, if the alignment was not requested in the alignment
+     * \attention Compiling this function will fail, if the alignment was not requested in the alignment
```

Comment:
As mentioned in the other PR, I don't fully get this, but we can discuss it after it has been merged.